src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxFactoryTests.cs

```diff
@@ -493,7 +493,7 @@ public void TestSpacingOnNullableIntType()
                 .NormalizeWhitespace();
 
             // no space between int and ?
-            Assert.Equal("class C\r\n{\r\n    int? P\r\n    {\r\n    }\r\n}", syntaxNode.ToFullString());
+            Assert.Equal("class C\r\n{\r\n    int? P { }\r\n}", syntaxNode.ToFullString());
```

Comment:
>{ } [](start = 53, length = 3)

Is this really a desired change?

Reply:
i'm personally ok with this. But i don't feel strongly.  Props with empty bodies are just not a hugely important case IMO. (Though auto-props def seem interesting).  Perhaps we should limit all changes here to members with *only* auto-props?

Reply:
That was my idea. I added the tests on non-auto properties for two reasons:
- @CyrusNajmabadi asked to add more tests :)
- I wanted to ensure the changes didn't modify the previous behavior.

This is also the reason for not adding tests on events but just checking they were not screwing up with my changes.
@AlekseyTs if you agree on this, I ask you to specifically mark the comments that don't need my review so that I can avoid spending time on those.
Thank you

Reply:
> Perhaps we should limit all changes here to members with only auto-props?

I do not have any strong objection for the current behavior. Just wanted to make sure that the behavior change is intentional and is ok from IDE point of view.

---
In reply to: [530576501](https://github.com/dotnet/roslyn/pull/49495#discussion_r530576501) [](ancestors = 530576501)

Reply:
> This is also the reason for not adding tests on events but just checking they were not screwing up with my changes.

It looks to me that the changes affect events as well. Is this intentional? I would be fine with that. Regardless, we should test events too and verify that they are affected/not affected as appropriate.

---
In reply to: [530585429](https://github.com/dotnet/roslyn/pull/49495#discussion_r530585429) [](ancestors = 530585429)

Reply:
@AlekseyTs @CyrusNajmabadi 
Before going on, I need to understand what do you prefer.
Since I am going to add more tests, what is your choice on this?
Do you want to use the same logic for properties as well or should I leave events to their old behavior (as I am doing now)?

```
event w e
{
    add => x += value;
    remove => x -= value;
}
```
or
```
event w e { add => x += value; remove => x -= value }
```

Reply:
>Before going on, I need to understand what do you prefer.
>Since I am going to add more tests, what is your choice on this?
>Do you want to use the same logic for properties as well or should I leave events to their old behavior (as I am doing now)?

The tests are needed regardless of the decision. Either way we have to verify the behavior after the change. The goal will be to either confirm, the behavior didn't change, or to confirm that it did change as expected. I think the normalization behavior should be consistent across events properties and indexers. However, for now, I suggest to add tests and capture the behavior as it is right now, then we can look at the tests and determine if something doesn't look/feel right.

---
In reply to: [531091605](https://github.com/dotnet/roslyn/pull/49495#discussion_r531091605) [](ancestors = 531091605)